hackathon_site/registration/forms.py

```diff
@@ -109,7 +109,23 @@ class Meta:
         }
 
     def __init__(self, *args, **kwargs):
+        self.user = kwargs.pop("user")
```

Comment:
What exactly does this do

Reply:
When the form is initalized, `user` is passed as a kwarg to it. This happens from the `get_form_kwargs` method of `ApplicationView` in registration/views.py. Since the actual base form class doesn't expect user as a kwargs, we pop it before initializing the superclass.